src/web/services/status-message.service.ts

```diff
@@ -47,12 +38,45 @@ export class StatusMessageService {
   showErrorMessage(message: string): void {
     this.showMessage({
       message,
-      color: 'danger',
+      color: 'snackbar-danger',
     });
   }
 
   private showMessage(message: StatusMessage): void {
-    this.alertEvent.next(message);
+    this.snackBar.open(message.message, '', {
+      duration: 5000,
+      verticalPosition: 'top',
+      panelClass: ['snackbar', message.color],
+    });
+  }
+
+  /**
+   * Shows a success message containing HTML on the page
+   */
+  showSuccessMessageTemplate(template: TemplateRef<any>): void {
+    this.showTemplate(template, 'snackbar-success');
+  }
+
+  /**
+   * Shows a warning message containing HTML on the page
+   */
+  showWarningMessageTemplate(template: TemplateRef<any>): void {
```

Comment:
Although it makes sense to include this in the event we have a need in the future, I feel we should not include it at this point.

Reason: We might not ever need it and if we ever need it in the future, we can add it in easily.